client/my-sites/email/email-management/home/email-plan.jsx

```diff
@@ -233,7 +233,7 @@ function EmailPlan( { domain, hideHeaderCake = false, selectedSite, source } ) {
 		}
 
 		return (
-			<VerticalNavItem { ...manageAllNavItemProps }>
+			<VerticalNavItem { ...manageAllNavItemProps } disabled={ ! hasSubscription || ! purchase }>
```

Comment:
That's correct. I think it's preferable that the `Manage all mailboxes` button matches the `View billing and payment settings` in disabled state rather than it being optimistically enabled. Let me know if you have any objections!